crates/bitwarden-vault/src/cipher/ssh_key.rs

```diff
@@ -0,0 +1,49 @@
+use bitwarden_crypto::{
+    CryptoError, EncString, KeyDecryptable, KeyEncryptable, SymmetricCryptoKey,
+};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct SshKey {
+    /// Ssh private key (ed25519/rsa) in unencrypted openssh private key format https://github.com/openssh/openssh-portable/blob/master/PROTOCOL.key
```

Comment:
Update the comments in this file to match whatever resolution from the comments in `crates/bitwarden-exporters/src/lib.rs`.

Reply:
Updated to use "SSH" and used markdown for the links in 0371ea1.